go/tools/builders/compilepkg.go

```diff
@@ -294,6 +294,9 @@ func compileArchive(
 		imports["unsafe"] = nil
 	}
 	if coverMode != "" {
+		if coverMode == "atomic" {
```

Comment:
Do I need to add this to compile.go? I have no access to coverMode in there though and it doesn't add cgo generated imports either.

Reply:
Nope, shouldn't be needed.

`compile.go` implements the `GoCompile` action. When coverage is needed, `GoCover` (in `cover.go`) is a separate action that comes first. These days, rules_go just emits `GoCompilePkg`, which does all the work to build a package in one action. Both `GoCompilePkg` and `GoCover` use `instrumentForCoverage` in `cover.go`, which already supports the mode.